src/main/java/org/minions/devfund/angela/Main.java

```diff
@@ -0,0 +1,59 @@
+package org.minions.devfund.angela;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * Manages execution.
+ */
+public final class Main {
+
+    /**
+     * Avoid constructor.
+     */
+    private Main() {
+    }
+
+    /**
+     * Executes the whack a mole game.
+     *
+     * @param args arguments to execute main.
+     */
+    public static void main(final String[] args) {
+        final int numAttempts = 50;
+        final int gridDimension = 10;
+        final WhackAMole whackAMole = new WhackAMole(numAttempts, gridDimension);
+        Random randomI = new Random();
+        Random randomJ = new Random();
+        final int molesQuantity = 10;
+        for (int i = 0; i < molesQuantity; i++) {
+            if (!whackAMole.place(randomI.nextInt(gridDimension), randomJ.nextInt(gridDimension))) {
+                i--;
+            }
+        }
+        for (int i = 0; i < numAttempts; i++) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)